spec/features/admin/invoices/index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'the admin invoices index' do
+  before(:each) do
+    @customer_1 = create(:customer)
+    @customer_2 = create(:customer)
+    @customer_3 = create(:customer)
+    @customer_4 = create(:customer)
+    @customer_5 = create(:customer)
+    @invoice_1 = create(:invoice, customer: @customer_1)
+    @invoice_2 = create(:invoice, customer: @customer_1)
+    @invoice_3 = create(:invoice, customer: @customer_2)
+    @invoice_4 = create(:invoice, customer: @customer_5)
+    @invoice_5 = create(:invoice, customer: @customer_5)
+  end
+
+  describe 'basics' do
+    it 'shows links to all invoices in the db' do
+      visit '/admin/invoices'
```

Comment:
do we want to standardize use of Route Helpers for this project?

Reply:
Yeah. let me change that real quick.